Gemfile.jruby-1.9.lock

```diff
@@ -94,7 +94,7 @@ GEM
     http_parser.rb (0.6.0-java)
     i18n (0.6.9)
     insist (1.0.0)
-    jar-dependencies (0.1.15)
+    jar-dependencies (0.2.1)
```

Comment:
makes sense, I don't think it should be runtime, at less from my research in #3735 .... I will report there.